src/Router.php

```diff
@@ -275,7 +275,7 @@ public static function send_header( $key, $value ) {
 		 * single SP before interpreting the field value or forwarding the
 		 * message downstream.
 		 */
-		$value = preg_replace( '/\s+/', ' ', $value );
+		$value = preg_replace( '/\s+/', ' ', $value ?? '' );
```

Comment:
Instead of preg-replacing an empty string, let's check if the string is _empty_ before preg-replacing.

```suggestion
		$value = ! empty( $value ) ? preg_replace( '/\s+/', ' ', $value ): '';
```